shared-src/docs-tech-guide.rst

```diff
@@ -463,11 +463,11 @@ Clone to local
 
 From your own fork of the repo on GitHub, 
 select the :guilabel:`Clone or download` button. 
-Copy the URI from the text box that opens up. 
+Copy the URL from the text box that opens. 
```

Comment:
URI is correct.

Reply:
For those following along, if the web address has a protocol in front of it, it's a URL. If it doesn't, it's a URI. In the case of this dialog, if you use clone with HTTPS, then it's a URL because the string GitHub shows starts with https://. If you clone with SSH, then it's a URI because there is no ssh://.

So @adammichaelwood is correct, but unecessarily pendatic.